.github/workflows/build-test-windows-x86.yml

```diff
@@ -243,9 +243,9 @@ jobs:
           opam env
 
           # opam pin --no-action add semgrep.dev .
-          echo "Pinning local packages..."
-          for f in opam/*.opam;
-            do opam pin --no-action add "$(basename "$f" .opam).dev" .; done;
+          # echo "Pinning local packages..."
```

Comment:
?

Reply:
This was not needed and it was slowing down the CI